newton/_src/viewer/gl/opengl.py

```
@@ -968,6 +1033,14 @@ def render(self, camera, objects, lines=None):
         self._view_matrix = self.camera.get_view_matrix()
         self._projection_matrix = self.camera.get_projection_matrix()
 
+        # Lazy-load environment map after a valid GL context is active
+        if self._env_path is not None and self._env_texture is None:
+            try:
+                self.set_environment_map(self._env_path)
+            except Exception:
+                pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)